docs/brand.md

```diff
@@ -0,0 +1,50 @@
+# 🎨 Our Brand
```

Comment:
Can you add Brand to the README.md table of contents? It's manually generated right now (sorry).